### PR TITLE
Fix an issue with markdown ordered list processing

### PR DESCRIPTION
**Pull request by testableapple**
### 🔗 Issue Links

Resolve https://github.com/GetStream/ios-issues-tracking/issues/850

### 🎯 Goal

- Fix an issue with markdown ordered list processing

#### Original text: 
```
The UPS Store

324 S Beverly st

Beverly Hills CA 90210. 
```

#### Then
```
The UPS Store

1. S Beverly st

Beverly Hills CA 90210. 
```

#### Now
```
The UPS Store

324 S Beverly st

Beverly Hills CA 90210. 
``` 

### 🧪 Manual Testing Notes

- Try an example from the showcase and any other relevant combinations
- Test ordered list

### ☑️ Contributor Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] This change follows zero ⚠️ policy (required)
- [x] This change should be manually QAed
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS)

### 🎁 Meme

gif

### CODE REVIEW

File: Sources/StreamChatUI/StreamSwiftyMarkdown/SwiftyMarkdown/SwiftyLineProcessor.swift [hunk 1]
```diff
@@ -250,7 +250,7 @@ class SwiftyLineProcessor {
     }
     
     func processOrderedListRegex(_ text: String) -> String {
-        let regex = try? NSRegularExpression(pattern: "^[0-9]+. ", options: .caseInsensitive)
+        let regex = try? NSRegularExpression(pattern: "^[0-9]+\\. ", options: .caseInsensitive)
```
Comment (laevandus) [inline Sources/StreamChatUI/StreamSwiftyMarkdown/SwiftyMarkdown/SwiftyLineProcessor.swift:253]: One unit-test for capturing this issue would be beneficial. E.g. if we change our markdown parser, then this issue would also get retested.

Comment (testableapple) [reply Sources/StreamChatUI/StreamSwiftyMarkdown/SwiftyMarkdown/SwiftyLineProcessor.swift:253]: I've added a snapshot test. Let me know if you think it's not enough 👍

Comment (laevandus) [reply Sources/StreamChatUI/StreamSwiftyMarkdown/SwiftyMarkdown/SwiftyLineProcessor.swift:253]: Sounds good!

**Review by laevandus — APPROVED** [review]

**Comment by laevandus** [comment]
This particular case is now rendered correctly, marking as QAd.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### ✅ Added
 - Low-level client support for Polls - creating different types of polls, adding and removing votes, comments, and more [#3220](https://github.com/GetStream/stream-chat-swift/pull/3220)
 
+## StreamChatUI
+### 🐞 Fixed
+- Fix an issue with markdown ordered list processing [#3234](https://github.com/GetStream/stream-chat-swift/issues/3230)
+
```
Comment (nuno-vieira) [inline CHANGELOG.md:13]: Make sure to move this to the #Upcoming section after merging with develop

**Review by nuno-vieira — APPROVED** [review]
LGTM! ✅
LGTM! ✅

**Pull request merged**